memgpt/openai_tools.py

```diff
@@ -102,10 +108,17 @@ async def wrapper(*args, **kwargs):
 
 @aretry_with_exponential_backoff
 async def acompletions_with_backoff(**kwargs):
-    azure_openai_deployment = os.getenv('AZURE_OPENAI_DEPLOYMENT')
-    if azure_openai_deployment is not None:
-        kwargs['deployment_id'] = azure_openai_deployment
-    return await openai.ChatCompletion.acreate(**kwargs)
+
+    # Local model

```

Comment:
Important caveat - embeddings calls are still getting routed to OpenAI (should be fine for now, can soft patch is a flag to turn off embeddings in archival and make it text search only)

Reply:
I can patch this into #96 after this PR is merged